nin/backend/generate/TemplateShaderLayer.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @constructor
+ */
+function TemplateLayer(config) {
+  this.config = config;
```

Comment:
It might be confusing to call this variable config. The variable that is passed in to the layer is an object containing, amongst other things, `startFrame`, `endFrame`, `config`, etc.